api/coreservice.go

```diff
@@ -371,6 +371,23 @@ func (core *coreService) ReadContract(ctx context.Context, callerAddr address.Ad
 	return res.Data, res.Receipt, nil
 }
 
+// SimulateExecution simulates the execution locally without broadcasting to the network
+func (core *coreService) SimulateExecution(callerAddr address.Address, sc action.Execution) ([]byte, *action.Receipt, error) {
```

Comment:
can you file an issue to replace `core.sf.SimulateExecution` with this new func?